### PR TITLE
Share button: local, external,  sketch

### LINKED ISSUE #178: share buttons
- [x] - open local url
- [x] - open external url
- [ ] - export to Sketch

### PR DESCRIPTION
**Pull request by awinogradov**
Close #178

### CODE REVIEW

File: src/components/dropdown/dropdown.post.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.dropdown {
```
Comment (Yeti-or) [inline src/components/dropdown/dropdown.post.css]: ?

File: src/components/icon/_action/icon_action_edit.svg [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (Yeti-or) [inline src/components/icon/_action/icon_action_edit.svg]: svgo?

Comment (awinogradov) [inline src/components/icon/_action/icon_action_edit.svg]: Yes) In future)

File: src/components/dropdown/dropdown.post.css (hunk 1 above)
Comment (awinogradov) [inline src/components/dropdown/dropdown.post.css]: I beleave)

File: src/components/share/share.browser.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  onSetMod: {
+    'js': {
+      'inited': function() {
+        const local = this.findBlockInside('local', 'link');
+        const external = this.findBlockInside('external', 'link');
+        const sketch = this.findBlockInside('sketch', 'link');
+
+        local.on('click', () => {
+          const project = this.findBlockOutside('editor').getProject();
```
Comment (Yeti-or) [inline src/components/share/share.browser.js]: Any ideas how do it better?

Comment (awinogradov) [inline src/components/share/share.browser.js]: Flux, redux? But why it is bad?

File: src/lib/screen.js [hunk 4]
```diff
@@ -95,6 +97,18 @@ exports.resolveIntractions = function resolveIntractions(scr, script) {
...
+exports.exportToSketch = function exportToSketch(scr, screenSource) {
+  const exec = bluebird.promisify(require('child_process').exec);
+
+  const filepath = join(
+    conf.get('app:directory'), '.temp', `${scr.name}.svg`
+  );
+
+  return fs
+    .outputFileAsync(filepath, screenSource)
+    .then(() => exec(`open -a Sketch ${filepath}`));
```
Comment (Yeti-or) [inline src/lib/screen.js]: 😆

Comment (awinogradov) [inline src/lib/screen.js]: Simple!)

File: src/components/share/share.browser.js (hunk 3 above)
Comment (Yeti-or) [inline src/components/share/share.browser.js]: dependency on upper component, not so good. But I think it's okay for today

**Comment by Yeti-or** [comment]
🆗

File: src/components/share/share.browser.js (hunk 3 above)
Comment (awinogradov) [inline src/components/share/share.browser.js]: We can implemnt redux, really! It is better in this case. One storage for all components. I can make issue:) ок?

Comment (Yeti-or) [inline src/components/share/share.browser.js]: yeah it seems appropriate

Comment (awinogradov) [inline src/components/share/share.browser.js]: https://github.com/theprotein/protein-dynamics/issues/199

**Pull request merged**